pytripgui/messages.py

```diff
@@ -32,7 +32,10 @@
         "Before continue, you should configure Trip98 paths in Settings/Trip98 Config"],
     "kernelSisPath": [
         "Given kernel has no sis path",
-        "Kernel selected by plan has no valid sis path"]
+        "Kernel selected by plan has no valid sis path"],
+    "noTargetRoiSelected": [
+        "No target ROI selected",
+        "Please select at least one TargetROI in 'Target' tab"]
```

Comment:
This text doesn't make sense, as by design user can specify exactly one TargetROI. Why then adding `at least one` ?